implementations/micrometer-registry-datadog/build.gradle

```diff
@@ -2,6 +2,7 @@ dependencies {
     api project(':micrometer-core')
 
     implementation 'org.slf4j:slf4j-api'
+    implementation 'com.datadoghq:java-dogstatsd-client'
```

Comment:
I've raised this to our team internally to evaluate the relevancy

Reply:
response that we're tracking internally, marking this as won't fix

> java-dogstatsd-client depends on jnr-unixsocket, which depends on jnr-posix.
> 
> Because we support Java 7+, we depend on an old version of jnr-unixsocket, 0.36.
> 
> This version depends on jnr-posix which[ has a vulnerability](https://security.snyk.io/vuln/SNYK-JAVA-COMGITHUBJNR-1570422) in a function that is not used by jnr-unixsocket and java-dogstatsd-client, so does NOT directly affect the client library.
> 
> The issue will be resolved for good when we drop Java 7 support and can upgrade dependency version.